### DISCUSSION TITLE
Command Line?

### DISCUSSION THREAD

**Discussion by SteveSartain** [dotnet/ResXResourceManager#481] [Ideas]
I'd love to be able to perform some of the functions of the app via command line. In my specific case I'm looking to identify any resource values that are orphaned. Through the UI I perform this by selecting "Detect Code References"  and identifying any that are 0

As part of our CI process, I would love to be able to output any orphans as warnings

**Comment by tom-englert** [comment]
Like this: https://github.com/dotnet/ResXResourceManager/blob/master/Documentation/Topics/Scripting.md?

**Reply by SteveSartain** [reply]
The scripting module doesn't check for unused references. I'm specifically trying to work out what resources are unused. Unless I've missed something?

**Reply by tom-englert** [reply]
You could provide a PR to add this feature
